lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -294,3 +294,16 @@ func convertStringsToNets(strs []string) []*cnet.IPNet {
 	}
 	return nets
 }
+
+// PrefixAndAppendSelector prefixes a new selector string with the given prefix and appends it to an existing selector string.
+func PrefixAndAppendSelector(currentSelector, newSelector, prefix string) string {
```

Comment:
Probably worth putting this (and the `parseSelectorAttachPrefix` and friends) into a new `selectors.go` file in this directory. Until now, they have only been used in policy rules (which is why they are in rules.go) but now they are used elsewhere it makes sense to give them their own home.

Reply:
Fixed